kitops/modelkit/kitfile.py

```diff
@@ -189,167 +152,33 @@ def load(self, path):
-    def package(self) -> Dict[str, Any]:
-        """
-        Get the package information.
+        self.build(**data)
 
-        Returns:
-            Dict[str, Any]: Package information.
-        """
-        return self._data["package"]
-
-    @package.setter
-    def package(self, value: Dict[str, Any]) -> None:
-        """
-        Set the package information.
-
-        Args:
-            value (Dict[str, Any]): Package information.
-        """
-        self._package_validator.validate(data=value)
-        self._data["package"] = value
-
-    @property
-    def code(self) -> List[Dict[str, Any]]:
-        """
-        Get the code section.
-
-        Returns:
-            List[Dict[str, Any]]: Code section.
-        """
-        return self._data["code"]
-
-    @code.setter
-    def code(self, value: List[Dict[str, Any]]) -> None:
-        """
-        Set the code section.
-
-        Args:
-            value (List[Dict[str, Any]]): Code section.
-        """
-        self._code_validator.validate(data=value)
-        self._data["code"] = value
-
-    @property
-    def datasets(self) -> List[Dict[str, Any]]:
-        """
-        Get the datasets section.
-
-        Returns:
-            List[Dict[str, Any]]: Datasets section.
-        """
-        return self._data["datasets"]
-
-    @datasets.setter
-    def datasets(self, value: List[Dict[str, Any]]) -> None:
-        """
-        Set the datasets section.
-
-        Args:
-            value (List[Dict[str, Any]]): Datasets section.
-        """
-        self._datasets_validator.validate(data=value)
-        self._data["datasets"] = value
-
-    @property
-    def docs(self) -> List[Dict[str, Any]]:
-        """
-        Get the docs section.
-
-        Returns:
-            List[Dict[str, Any]]: Docs section.
-        """
-        return self._data["docs"]
-
-    @docs.setter
-    def docs(self, value: List[Dict[str, Any]]) -> None:
-        """
-        Set the docs section.
-
-        Args:
-            value (List[Dict[str, Any]]): Docs section.
-        """
-        self._docs_validator.validate(data=value)
-        self._data["docs"] = value
-
-    @property
-    def model(self) -> Dict[str, Any]:
-        """
-        Get the model section.
-
-        Returns:
-            Dict[str, Any]: Model section.
-        """
-        return self._data["model"]
-
-    @model.setter
-    def model(self, value: Dict[str, Any]) -> None:
-        """
-        Set the model section.
-
-        Args:
-            value (Dict[str, Any]): Model section.
-        """
-        self._model_validator.validate(data=value)
-        self._data["model"] = value
-
-    def to_yaml(self, suppress_empty_values: bool = True) -> str:
+    def to_yaml(self, no_empty_values: bool = True) -> str:
         """
         Serialize the Kitfile to YAML format.
 
         Args:
-            suppress_empty_values (bool, optional): Whether to suppress
+            no_empty_values (bool, optional): Whether to suppress
```

Comment:
This will break anyone doing passing `suppress_empty_values` to `to_yaml()`. Since this is just renaming, I would suggest to revert and go back to `suppress_empty_values`.

Reply:
Reverted